helm/adelphi/templates/nosqlbench-jobs.yaml

```diff
@@ -29,21 +29,33 @@ spec:
       - name: secret-volume
         mountPath: /secret
         readOnly: true
-      - name: workload
-        mountPath: /config/workload.yaml
-        subPath: workload.yaml
+      - name: workspace
+        mountPath: /workspace
       - name: results-pv
         mountPath: /results
+    initContainers:
+    - name: extract-schema
+      image: python:3.10.0b1-alpine3.13
+      command: ["sh", "-c", "-e"]
+      args:
+        - pip install adelphi;
+          adelphi --hosts="{{ .Values.source.clusterName }}-{{ .Values.source.dc }}-service" --username="{{ .Values.source.clusterName }}-superuser" --password="`cat /secret/password`" --keyspaces="{{ .Values.gemini_schema_keyspace }}" --no-anonymize export-nb > /workspace/nosqlbench_source.yaml;
```

Comment:
I'm a little worried about bringing this in until we have full support for all the column types (i.e. https://github.com/datastax/adelphi/issues/100).

Reply:
@absurdfarce What is in master right now doesn't even use the provided schema, so this is obviously better. We have other tickets to handle failures/logging scenarios.

If there's still a concern, let's just make our docs more clear, but let's not hold the feature back.